tutorials/ai-core-and-LP-provisioning-GA/ai-core-and-LP-provisioning-GA.md

```diff
@@ -0,0 +1,118 @@
+---
+parser: v2
+auto_validation: true
+time: 20
+tags: [ tutorial>license, tutorial>beginner, topic>artificial-intelligence, topic>provisioning, software-product>sap-ai-core ]
+primary_tag: software-product>sap-ai-core
+author_name: Dhrubajyoti Paul
+author_profile: https://github.com/dhrubpaul
+---
+
+# Use Boosters for Free Tier Use of SAP AI Core and SAP AI Launchpad
+
+## Prerequisites
+- A BTP global account
+
+## You will learn
+- How to access your SAP BTP global account
+- How to use boosters to get started with SAP AI Core and SAP AI Launchpad
+
+## About SAP AI Core and SAP AI Launchpad
+SAP AI Core and SAP AI Launchpad are services which you can link to your BTP global account. SAP AI Core offers a powerful AI runtime which is natively integrated with SAP AI Launchpad. The launchpad offers an easy-to-use interface to manage AI workflow administration, processes, and tasks.
+
+## How Does Free Tier Work?
+Both services are available as either free tier or standard (paid) plans. If you choose **free tier** plans, you can try out SAP AI Core and SAP AI Launchpad for free.
+
+After testing and evaluation, you can easily upgrade free tier to paid plans without losing your data or work. Note that creating an instance using a paid plan will incur costs.
+
+Boosters are available for both services to help you get started. Boosters are a set of guided interactive steps that enable you to select, configure, and consume services on SAP BTP. You can use boosters for both free tier and paid plans. In this tutorial, we'll use the boosters to set you up with free tier plans.
+
+For more information, see [Free Tier (SAP AI Core)](https://help.sap.com/docs/AI_CORE/2d6c5984063c40a59eda62f4a9135bee/4533adc472074698b355c70f04b2cf49.html?version=CLOUD) and [Free Tier (SAP AI Launchpad](https://help.sap.com/docs/AI_LAUNCHPAD/92d77f26188e4582897b9106b9cb72e0/87e4fb191e3746c0850f76642da96871.html?version=CLOUD)).
+
+## Pre-read
+To find out more about free tier service plans, visit the [SAP Discovery Center](https://discovery-center.cloud.sap/#/serviceCatalog?provider=all&regions=all&category=freetierservices) and explore [SAP AI Launchpad](https://discovery-center.cloud.sap/serviceCatalog/sap-ai-launchpad?region=all) and [SAP AI Core](https://discovery-center.cloud.sap/serviceCatalog/sap-ai-core?region=all).
+
+> Remember, only community support is available for free tier service plans and these are not subject to SLAs.
+
+---
+
+### Provision SAP AI Core in your global account
+Go to the BTP Control Center and select your global account. Click `+`. 
+
+<!-- border -->![image](img/BTPControlCentre.png)
```

Comment:
For this and other images, we recommend that you use text to describe the image (instead of "image") in case a user is using a screen reader.  This supports our accessibility standards.

Reply:
I have made the requested changes